src/modules/control_allocator/ControlAllocation/ControlAllocation.cpp

```diff
@@ -52,6 +52,10 @@ ControlAllocation::setEffectivenessMatrix(
 	_control_trim = _effectiveness * _actuator_trim;
 	_num_actuators = num_actuators;
 
+	// make sure unused actuators are initialized to min
+	for (int i = num_actuators; i < NUM_ACTUATORS; ++i) {
+		_actuator_sp(i) = _actuator_min(i);
```

Comment:
Good catch. 
However there is again the assumption that the last actuators are unused.
Also, I think _actuator_trim is more appropriate.

Reply:
We can use trim for now, but eventually I intend to use NaN to ensure outputs are disabled.